nova/nova/permissions.py

```diff
@@ -13,11 +13,34 @@ def has_permission(self, request, view):
         return bool(request.user and request.user.is_authenticated)
 
 
+class IsAuthenticated(permissions.BasePermission):
```

Comment:
AFAIK, there is already an IsAuthenticated class

Reply:
I think I was doing experiments with this class, will remove that.